python/GafferUI/PresetsPlugValueWidget.py

```diff
@@ -58,6 +58,11 @@ def __init__( self, plugs, **kw ) :
 		cont.addChild( self.__menuButton )
 
 		self.__customValuePlugWidget = GafferUI.PlugValueWidget.create( plugs, useTypeOnly=True )
+		if type( self.__customValuePlugWidget ) == GafferUI.ConnectionPlugValueWidget and len( self.getPlugs() ) and next( iter( self.getPlugs() ) ).children():
+			# If we couldn't find a meaningful plug widget type, and the plug has children, try using
+			# a layout widget to show child plugs when in "custom" mode
+			self.__customValuePlugWidget = GafferUI.LayoutPlugValueWidget( plugs )
```

Comment:
I agree it looks totally wrong ( I think it would be fine not to add any additional indent if there was an easy way to accomplish that ), but am tempted to punt it.  Given how rare it will hopefully be that someone needs to edit this, maybe it's OK they will need an awkwardly wide window.

Would be worth improving if this gets used in more places though ( or if it allowed us to replace FormatPlugValueWidget )

Reply:
Cool, let's punt it then.